src/ansible_navigator/_yaml.py

```diff
@@ -45,10 +51,16 @@ def human_dump(obj: Any, filename: str = None, fmode: str = "w") -> Union[str, N
 
 class HumanDumper(Dumper):
     # pylint: disable=too-many-ancestors
-    """for block scalar for multiline"""
+    """Dump a multiline in a human readable format."""
```

Comment:
This one is the exact opposite of my previous comment — it should describe a static thing/concept/object. By itself, a class instance does not do anything (unless it implements a `__call__()` method, that is; but even then, I'd put the action in the dunder-call docstring, not the class docstring), it does not dump a multiline; its methods do the dumping but merely initializing this object does not have side-effects as it currently implies.

*Hint:* for classes/constants/vars/attrs/properties/modules/packages, describe what they are, not what they "do". If there's a side effect of doing something during the initialization, something is seriously wrong with that design.

```suggestion
    """An implementation of YAML the dumping protocol

    It is designed with human readability in mind and its methods
    emit multiline string representation.
    """
```
(not sure how precise this is but I hope you got the idea)

Reply:
got it